apps/strapi-cms/src/api/webinar/content-types/webinar/lifecycles.ts

```diff
@@ -36,11 +64,125 @@ const validateDates = (event: IWebinarEvent): boolean => {
   return true;
 };
 
+const validateSlug = async (event: IWebinarEvent): Promise<boolean> => {
+  const { id } = event.params.data;
+  if (!id) {
+    throw new errors.ApplicationError('Webinar id not found');
+  }
+  const previousWebinar = await strapi.db
```

Comment:
Why is it called previous and not just webinar?

Reply:
To declare explicitly that the query is taking the webinar value before the update (validateSlug is called on beforeUpdate hook), to check if the slug is going to change during the update